src/operator/quantization/quantized_conv.cc

```diff
@@ -126,6 +126,15 @@ bool QuantizedConvStorageType(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+
+void QuantizedConvForward(const nnvm::NodeAttrs& attrs,
+                             const OpContext &ctx,
+                             const std::vector<TBlob> &in_data,
+                             const std::vector<OpReqType> &req,
+                             const std::vector<TBlob> &out_data) {
+    LOG(FATAL) << "NOT SUPPORTED";
```

Comment:
please use better error message here: Not supported for MXNet built without MKLDNN. Please install MKLDNN enabled MXNet.

Reply:
@anirudh2290 Its just WIP i will update it to proper message. Want to ensure that the code behaviour is as expected before finalizing the Error Message. :)